client/src/partials/creditor/creditor.html

```diff
@@ -62,7 +62,7 @@
               </div>
 
               <div class="form-group">
-                <label for="ad1">{{ "SUPPLIER.ADDR_1" | translate }}</label>
+                <label for="ad1" class="control-label required">{{ "SUPPLIER.ADDR_1" | translate }}</label>
```

Comment:
You don't need the `control-label` class here.  The `control-label` tag is used with `form-horizontal`, or when using form validation (which this module does not use).  If you would like, I can show you how to implement form validation tomorrow.